client/my-sites/plans-features-main/test/plan-type-selector.jsx

```diff
@@ -1,14 +1,21 @@
+/** @jest-environment jsdom */
 jest.mock( 'i18n-calypso', () => ( {
 	localize: ( Component ) => ( props ) => <Component { ...props } translate={ ( x ) => x } />,
 	numberFormat: ( x ) => x,
 	translate: ( x ) => x,
 	useTranslate: jest.fn( () => ( x ) => x ),
 } ) );
+jest.mock( '@automattic/components', () => ( {
+	Popover: () => <div />,
+} ) );
 
 import { PLAN_FREE } from '@automattic/calypso-products';
-import { shallow } from 'enzyme';
+import { screen } from '@testing-library/react';
+import { renderWithProvider } from 'calypso/test-helpers/testing-library';
 import PlanTypeSelector, { CustomerTypeToggle } from '../plan-type-selector';
 
+const render = ( el, options ) => renderWithProvider( el, { ...options } );
```

Comment:
What's the wrapper useful for?

Reply:
It will provide Redux context and store so the components can access the Redux state tree.

Also, I find this way of expressing it (aliasing it to `render`) pretty nice since it helps us keep the same format for writing tests. The only downside could be that we're concealing that we're adding on top of the default `render()` function, but it's easy to trace anyway.

Reply:
I usually add missing reducers. In this case we don't need any more of them though. We could just alias it right on import instead of using a function decl like this. No strong opinion, happy to proceed with either.